src/units/UnitsLiteralTreeAnnotator.java

```diff
@@ -2,25 +2,25 @@
 
 import org.checkerframework.framework.qual.LiteralKind;
 import org.checkerframework.framework.type.AnnotatedTypeFactory;
-import org.checkerframework.framework.type.treeannotator.ImplicitsTreeAnnotator;
+import org.checkerframework.framework.type.treeannotator.LiteralTreeAnnotator;
 import units.representation.UnitsRepresentationUtils;
 
-/** Common base class for ImplicitsTreeAnnotator. */
-public abstract class UnitsImplicitsTreeAnnotator extends ImplicitsTreeAnnotator {
+/** Common base class for LiteralTreeAnnotator. */
+public abstract class UnitsLiteralTreeAnnotator extends LiteralTreeAnnotator {
 
     UnitsRepresentationUtils unitsRepUtils = UnitsRepresentationUtils.getInstance();
 
-    // Programmatically set the qualifier implicits
-    public UnitsImplicitsTreeAnnotator(AnnotatedTypeFactory atf) {
+    // Programmatically set the qualifier for literals
+    public UnitsLiteralTreeAnnotator(AnnotatedTypeFactory atf) {
         super(atf);
 
-        // set BOTTOM as the implicit qualifier for null literals
+        // set BOTTOM as the literal qualifier for null literals
```

Comment:
Couldn't these also be done declaratively?

Reply:
The type checking mode and the inference mode have different literals defaults.